cmd/node/main.go

```diff
@@ -86,62 +86,70 @@ VERSION:
 
 	// genesisFile defines a flag for the path of the bootstrapping file.
 	genesisFile = cli.StringFlag{
-		Name:  "genesis-file",
-		Usage: "The node will extract bootstrapping info from the genesis.json",
+		Name: "genesis-file",
+		Usage: "The `filepath` for the genesis file in JSON format. This file should contain initial data to bootstrap" +
+			" from, such as initial balances for accounts.",
```

Comment:
The path to the genesis file. This is a JSON file which contains initial data ...